ts/test/test-time-profiler.ts

```diff
@@ -73,15 +73,20 @@ describe('Time Profiler', () => {
       initialContext['aaa'] = 'bbb';
 
       let endTime = 0n;
-      time.stop(false, (context: TimeProfileNodeContext) => {
+      time.stop(false, (context?: TimeProfileNodeContext) => {
+        assert.ok(context !== null, 'Context should not be null');
         if (!endTime) {
           endTime = BigInt(Date.now()) * 1000n;
         }
-        assert.deepEqual(context.context, initialContext, 'Unexpected context');
-        assert.ok(context.timestamp >= startTime);
-        assert.ok(context.timestamp <= endTime);
+        assert.deepEqual(
+          context!.context,
+          initialContext,
+          'Unexpected context'
+        );
+        assert.ok(context!.timestamp >= startTime);
+        assert.ok(context!.timestamp <= endTime);
         checked = true;
-        return {};
+        return {...context!.context};
```

Comment:
Weird that we didn't do that before. I guess nothing checks it?

Reply:
I had to do this, otherwise the sample is bundled in the unlabelled hits, and `generateLabels` is called a second time without argument, which would cause the test to fail.